core/ffmpeg/ffmpeg.go

```diff
@@ -27,7 +27,9 @@ func New() FFmpeg {
 }
 
 const (
-	extractImageCmd = "ffmpeg -i %s -an -vcodec copy -f image2pipe -"
+	// OGG cover uses "Theora", which is not supported by the browser's image tag.
+	// However, we can tell ffmpeg to perform the image conversion for us.
+	extractImageCmd = "ffmpeg -i %s -an -c:v mjpeg -f image2pipe -"
```

Comment:
I don't know ffmpeg commands super well but does this convert all covers to jpeg, or only theora? I assume we don't want to convert png to jpg to avoid quality loss since png is supported everywhere

Reply:
It will always converted to jpg. To conditionally copy/convert, we need to know the video stream content beforehand and decide which codec to use (`copy` vs `mjpeg`).

Reply:
Perhaps it should be rewritten then to try the mjpeg extraction only if the copy extraction fails? I think it's a regression of existing behavior if navidrome previously extracted embedded PNG art without converting it and now it won't.

Reply:
Make sense. The copy won't fail in ogg's case though - it extracts and serves the "raw" theora which was not identifiable by Chrome.

We could cache the video stream type during probing, or run the probing again on the fly when we need to extract the image.

I can give this a try tomorrow.

Reply:
Changed to detect image streams.

In the case of jpg/png/webp, it will extract as it is. Otherwise, it will be converted to jpg.